src/Profiling.cpp

```diff
@@ -30,6 +72,8 @@ class InjectProfiling : public IRMutator {
 
     string pipeline_name;
 
+    bool in_fork = false, in_parallel = false, in_leaf_task = false;
```

Comment:
ubernit: this is an unusual formatting for our code; we almost always put member var declarations one-per-line.